modules/cql/test/blaze/elm/compiler/comparison_operators_test.clj

```diff
@@ -342,6 +357,32 @@
       {:type "Null"} #elm/quantity [1 "s"] false
       #elm/quantity [1 "s"] {:type "Null"} false))
 
+  (testing "Ratio"
+    (are [x y res] (= res (core/-eval (c/compile {} (elm/equivalent [x y])) {} nil nil))
```

Comment:
Can you please convert all tests inside the equivalent test into the form with `compile-binop` like it is done in the equal test. That's a bit more than doing this just for ratio but a good thing to do according the boy-scout rule.